src/microbe_stage/MicrobeColony.cs

```diff
@@ -117,5 +148,19 @@ public void AddToColony(Microbe microbe, Microbe master)
         microbe.ColonyChildren = new List<Microbe>();
 
         ColonyMembers.ForEach(m => m.OnColonyMemberAdded(microbe));
+
+        hexCountDirty = true;
+    }
+
+    private void UpdateHexCount()
+    {
+        hexCount = Master.EngulfSize;
+        ingestedSizeCount = Master.IngestedSizeCount;
+
+        foreach (var member in ColonyMembers)
```

Comment:
I'm 99% sure that the colony members include the master as well, so it is double counted here.

Reply:
I think that should be changed at some point, it's not intuitive at all and there's zero documentation mentioning it but I won't do it here as it will further complexify this PR.

Reply:
I think it's quite intuitive for many cases. We don't take full advantage of that, but if we add things that colony members can trigger then running code on all members is useful. I think currently on the state setting uses that. There it's very useful to be able to say "for all colony members set state to x", and that would be a lot more complicated if the colony master was handled differently. As you found this confusing it probably warrants a comment that "yes, list of group members, also includes the group leader".